### DISCUSSION TITLE
Potential bug issues

### DISCUSSION THREAD

**Discussion by geraldsamosir** [webpack/webpack#18286] [Q&A] [answered]
i want  to confirm code on [line 255](https://github.com/webpack/webpack/blob/main/lib/DefinePlugin.js#L255C24-L255C30) that is that code write like this

```js
	if (typeof code === "bigint") {  // here the code that i mention
			return runtimeTemplate.supportsBigIntLiteral()
				? `${code}n`
				: `BigInt("${code}")`;
		}
		return code + "";
	};

```

in javascript or EcmaSctipt  and  using bable transpailer that i know no data type "bigint" just  modern data type like 
object  number string  array. if that true i would recomended to use number because when i try to check on the browser and node.js on version on top `v20.0.0` is that posible to adjusted?

**Comment by alexander-akait** [comment]
Sorry, it is not related to babel, we are about bigints

**Reply by geraldsamosir** [reply]
i mean that no data type 
```js
typeof code
```
there are no "bigint"

**Reply by alexander-akait** [reply, marked as answer]
You can write `MY_CUSTOM_CODE: 1n` and it is a bigint
